vignettes/explanation/box-modules.Rmd

```diff
@@ -60,7 +60,7 @@ say_bye <- function(name) {
 Both `say_hello()` and `say_bye()` can be exported from `app/logic/messages.R`.
 
 ```r
-box::use(app/logic/messages[say_hello, say_bye])
+box::use(app/logic/messages[say_hello, say_bye],)
```

Comment:
I'd suggest to either transform this `box::use()` to a multi-line statement, or to omit the trailing comma. (The style guide only encourages trailing commas; in a single-line `box::use()` it's neither useful nor pretty.)

There's another single-line statement in line 119.